src/main/__test__/kube-auth-proxy.test.ts

```diff
@@ -223,34 +223,34 @@ describe("kube auth proxy tests", () => {
       await proxy.run();
       listeners.emit("error", { message: "foobarbat" });
 
-      expect(mockBroadcastIpc).toBeCalledWith("cluster:foobar:connection-update", { message: "foobarbat", isError: true });
+      expect(mockBroadcastIpc).toBeCalledWith("cluster:connection-update", "foobar", { message: "foobarbat", isError: true });
```

Comment:
Nit: we could prefer something more informative than "foobar" for naming in testing in general. 

Eg. here instead of "foobar", we could have "some-cluster-id". This can make a big difference when the scenarios grow less trivial.